app/chat_gpt_client.py

```diff
@@ -3,34 +3,46 @@
 import os
 import logging
 from typing import List
+import agentops
+from dotenv import load_dotenv
 
+# Load environment variables
+load_dotenv()
+
+# Initialize AgentOps
+AGENTOPS_API_KEY = os.getenv("AGENTOPS_API_KEY")
+if not AGENTOPS_API_KEY:
+    raise ValueError("AGENTOPS_API_KEY not found in environment variables")
+
+agentops.init(AGENTOPS_API_KEY)
```

Comment:
This initialization should happen in the `app.py` only, since it only needs to happen when the app is bootstrapped.
```suggestion
```